src/cli/commands/run.js

```diff
@@ -58,7 +62,7 @@ export async function run(config: Config, reporter: Reporter, flags: Object, arg
       cmdHints[cmd] = pkgScripts[cmd] || '';
     }
 
-    Object.assign(scripts, pkg.scripts);
+    Object.assign(scripts, pkg.scripts, shellCommands);
```

Comment:
`shellCommands` should probably be put before `pkg.scripts` so that user scripts can override the default mapping